src/main/java/com/openlattice/edm/type/PropertyType.java

```diff
@@ -89,7 +83,11 @@ public PropertyType(
         this.piiField = piiField.orElse( false );
         this.multiValued = multiValued.orElse( true );
         this.analyzer = analyzer.orElse( Analyzer.STANDARD );
-        this.postgresIndexType = postgresIndexType.orElse( IndexType.BTREE );
+        if ( EdmPrimitiveTypeKind.Binary == this.datatype ) {
```

Comment:
Thinking now that this might be the wrong place for this 😕

Reply:
Why?

Reply:
Had a conversation in here about it: https://github.com/openlattice/datastore/pull/384